src/ec/packages/ec-component-card/card.story.js

```diff
@@ -0,0 +1,112 @@
+import { storiesOf } from '@storybook/html';
+import { withKnobs, text, array } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import cardDocs from './docs/card.md';
+
+import card from './card.html.twig';
+
+storiesOf('Components/Card', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'card',
+    () => {
+      const tagsArray = array(
+        'Tags (comma separated)',
+        ['Tag 1', 'Tag 2', 'Tag 3'],
+        ','
+      );
+      const infosArray = array(
+        'Infos (comma separated)',
+        ['2018/10/22', 'Luxembourg'],
+        ','
+      );
+      const metaArray = array(
+        'Meta (comma separated)',
+        ['Meta 1', 'Meta 2', 'Meta 3'],
+        ','
+      );
+      const tags = tagsArray.map(tag => ({
+        label: tag,
+        path: '/example',
+      }));
+
+      const infos = infosArray.map((info, key) => ({
+        label: info,
+        icon: {
+          type: 'general',
+          name: () => {
+            if (key === 0) {
+              return 'calendar';
+            }
+
+            if (key === 1) {
+              return 'location';
+            }
+
+            return 'faq';
+          },
+          path: defaultSprite,
```

Comment:
open question here: why don't you use 'static/icons.svg' instead of loading the icons from ECL?
as far as I know it would work too, and that would remove an extra devDependency

Reply:
In all previous components, we always gave URLs to icons from a specific resource, so that you can easily change URLs easily and globally - I think so. We did it for the first time, probably in EC Icon, and in this way I'm duplicating it.
Do you want to set a specific URL to the icons in this case?

Reply:
it's fine as it is